distributed/worker.py

```diff
@@ -26,10 +26,9 @@
 from datetime import timedelta
 from inspect import isawaitable
 from pickle import PicklingError
-from typing import TYPE_CHECKING, Any, ClassVar, Container
+from typing import TYPE_CHECKING, Any, ClassVar, Container, Literal
 
 if TYPE_CHECKING:
-    from typing_extensions import Literal
```

Comment:
Is `if TYPE_CHECKING:` still needed here? It appears to be dropped elsewhere

Reply:
The other imports are python version independent and only needed for type checking but not for runtime.

Reply:
it protects what would otherwise be a circular import (which mypy deals with).